test/tests/functional/pbs_mom_walltime.py

```diff
@@ -147,17 +147,26 @@ def test_suspend_time_not_counted_in_walltime(self):
         self.server.manager(MGR_CMD_SET, MGR_OBJ_SERVER,
                             {'scheduling': 'True'})
 
-        time.sleep(15)
+        time.sleep(30)
 
         # Used walltime should remain the same
         self.server.expect(JOB, {'resources_used.walltime': walltime}, op=EQ,
-                           id=jid1, extend='x')
+                           id=jid1)
 
         self.server.sigjob(jobid=jid1, signal="resume")
         self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
         time.sleep(10)
         self.server.expect(JOB, {'resources_used.walltime': walltime}, op=GT,
```

Comment:
remove this sleep and check that walltime is greater value. the check you have added for the finished job is sufficient.